src/settings.ts

```diff
@@ -74,57 +74,55 @@ export class Settings {
    * @return {string} The bearer token.
    */
   public static getBearerToken(): string {
-    return Settings.readFileContent("~/.cloco/config/cloco_token");
+    return Settings.readFileContent(`${process.env.HOME}/.cloco/config/cloco_token`);
```

Comment:
That's a good idea actually, as I know that it's different on win32.  

function getUserHome() {
  return process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
}

[http://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way](http://stackoverflow.com/questions/9080085/node-js-find-home-directory-in-platform-agnostic-way)